### PR TITLE
VB-650 Refactor Integration Test and Unit tests

### PR DESCRIPTION
**Pull request by AledWEvans**
## What does this pull request do?

Refactor of test packages and some file names

## What is the intent behind these changes?

To improve readability and maintainability

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/config/ClientTrackingConfigurationTest.kt [hunk 1]
```diff
@@ -16,7 +16,7 @@ import org.springframework.mock.web.MockHttpServletResponse
 import org.springframework.test.context.ActiveProfiles
 import org.springframework.test.context.ContextConfiguration
 import org.springframework.test.context.junit.jupiter.SpringExtension
-import uk.gov.justice.digital.hmpps.visitscheduler.helper.JwtAuthHelper
+import uk.gov.justice.digital.hmpps.visitscheduler.test_setup.helper.JwtAuthHelper
```
Comment (AledWEvans) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/config/ClientTrackingConfigurationTest.kt:19]: Not sure about this name, have you a better Idea? I just wanted to separate the tests from the testing config classes

File: src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/config/ClientTrackingConfigurationTest.kt [hunk 2]
```diff
@@ -43,11 +43,17 @@ class ClientTrackingConfigurationTest {
...
   @Test
   fun shouldAddClientIdAndUserNameToInsightTelemetry() {
+
+    // Given
     val token = jwtAuthHelper.createJwt("bob")
     val req = MockHttpServletRequest()
     req.addHeader(HttpHeaders.AUTHORIZATION, "Bearer $token")
     val res = MockHttpServletResponse()
+
+    // When
```
Comment (AledWEvans) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/config/ClientTrackingConfigurationTest.kt:53]: Used the Given/When/Then convention

File: src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/config/ClientTrackingConfigurationTest.kt (hunk 1 above)
Comment (philip-milne) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/config/ClientTrackingConfigurationTest.kt:19]: If your not sure then just leave it as it was.

File: src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/config/ClientTrackingConfigurationTest.kt (hunk 2 above)
Comment (philip-milne) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/config/ClientTrackingConfigurationTest.kt:53]: We shouldn't need these comments in the tests, most of the tests should be following this convention

Comment (AledWEvans) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/config/ClientTrackingConfigurationTest.kt:53]: The comments are a standard convention with testing!

File: src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/config/ClientTrackingConfigurationTest.kt (hunk 1 above)
Comment (AledWEvans) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/config/ClientTrackingConfigurationTest.kt:19]: yes not sure will talk about it laters

**Review by philip-milne — DISMISSED** [review]
LGTM

**Pull request merged**